Cluster/How to configure APIM for Service Fabric managed cluster.md

```diff
@@ -0,0 +1,695 @@
+# How to configure APIM Service Fabric Managed Cluster Service Connection
+
+The steps below describe how to configure [Azure API Management](https://learn.microsoft.com/azure/api-management/) (APIM) to route traffic to a back-end service in a Service Fabric managed cluster using PowerShell. For unmanaged clusters, use [Integrate API Management with Service Fabric in Azure](https://learn.microsoft.com/azure/service-fabric/service-fabric-tutorial-deploy-api-management).
+
+Service Fabric Managed Clusters provision and manage the 'server' certificate including the rollover process before certificate expiration.
+There is currently no notification when this occurs.
+APIM service connections use X509 Certificate authentication requiring the configuration of the server certificate thumbprint.
+When the certificate is rolled over, the APIM connection will fail to connect to cluster causing applications to fail.
+
+## Requirements
+
+- Service Fabric managed cluster deployed using an existing external virtual network. See [Bring your own virtual network](https://learn.microsoft.com/azure/service-fabric/how-to-managed-cluster-networking#bring-your-own-virtual-network) in [Configure network settings for Service Fabric managed clusters](https://learn.microsoft.com/azure/service-fabric/how-to-managed-cluster-networking) for additional  information.
+- [Azure API Management](https://learn.microsoft.com/azure/api-management/).
+- Azure Key vault with certificate.
+
+## Process
+
+1. First, use New-AzResourceGroup to create a resource group to host the virtual network. Run the following code to create a resource group named TestRG in the eastus Azure region.
+
+    ```powershell
+    $resourceGroupName = 'testrg'
+    $location = 'eastus'
+    New-AzResourceGroup -Name $resourceGroupName -location $location
+    ```
+
+1. Create a Network Security Group for APIM
+
+    ```powershell
+    $networkSecurityGroup = New-AzNetworkSecurityGroup -Name 'vnet-apim-nsg' -ResourceGroupName $resourceGroupName  -Location $location
+    ```
+
+1. Configure NSG rules for APIM (Management endpoint for Azure portal and PowerShell)
```

Comment:
for management endpoint in APIM via Azure Portal or PowerShell?

Reply:
fixed